### DISCUSSION TITLE
Where can I see an example of Free-Text Search (q parameter)

### DISCUSSION THREAD

**Discussion by chiarch84** [radiantearth/stac-browser#379] [Q&A]
Dear @m-mohr  I looked in the details of release 3.1.0 and saw that STAC Browser should now support Free-Text Search (q parameter) is there an example of this that I can see somewhere? Is it supposed to be configured somehow or it just needs the relative conformance class to be set?
Thanks for you help and work!

**Comment by m-mohr** [comment]
Closing as duplicate of  #378

**Comment by chiarch84** [comment]
Dear @m-mohr I think both issues were closed as duplicates but none of them was answered. Can you please point me out a STAC catalog where I can test the text search? In particular I would be interested in the Collections text search. Thanks!

**Reply by m-mohr** [reply]
See https://github.com/radiantearth/stac-browser/discussions/378#discussioncomment-7185159

**Reply by chiarch84** [reply]
Thank you very much, I could not find anymore that answer!
